playbooks/appsemblerPlaybooks/eucalyptus_pro_server.yml

```diff
@@ -58,6 +59,7 @@
       when: "INSTALL_DEMO_DATA"
     - oauth_client_setup
     - forum
+    - postgresql_aggregation
```

Comment:
Do we want to make this role conditional on `COMMON_ENVIRONMENT == "prod"` so this isn't installed on staging?